core/src/main/java/tech/tablesaw/io/DataFrameReader.java

```diff
@@ -115,4 +117,14 @@ public Table db(ResultSet resultSet, String tableName) throws SQLException {
     public Table html(String url) throws IOException {
         return new HtmlTableReader().read(url);
     }
+    
+    // xlsx
+    
+    public Table[] xlsx(XlsxReadOptions options) throws IOException {
```

Comment:
I wonder if it might be nicer to return a `List<Table>`. I find `List` usually easier to work with than `Array`